plugins/woocommerce-admin/client/core-profiler/style.scss

```
@@ -122,3 +124,70 @@
 		}
 	}
 }
+
+.woocommerce-profiler-business-location {
+	.woocommerce-profiler-heading {
+		margin-top: 72px;
+	}
+	.woocommerce-profiler-go-to-mystore__button {
+		width: 400px;
+		display: flex;
+		justify-content: center;
+	}
+	.components-base-control__field {
+		height: 40px;
+	}
+	.woocommerce-profiler-select-control__country {
+		width: 400px;
+		margin-bottom: 32px;
+		.woocommerce-select-control__control {
+			height: 40px;
+		}
+		#woocommerce-select-control__listbox-0 {
+			top: 40px !important;
+		}
+	}
+	.woocommerce-select-control__control-icon {
+		display: none;
+	}
+
+	.woocommerce-select-control__control.is-active {
+		.components-base-control__label {
+			display: none;
+		}
+	}
+
+	.woocommerce-select-control.is-searchable
+		.woocommerce-select-control__control-input {
+		margin: 0;
+		padding: 0;
+	}
+
+	.woocommerce-select-control.is-searchable .components-base-control__label {
+		left: 13px;
+	}
+}
+
+.woocommerce-profiler-loader {
+	width: 100%;
+	height: 100%;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	flex-direction: column;
+	position: absolute;
+
+	h1 {
+		padding: 0;
+		margin: 58px 0 0 0;
+		font-size: 26px;
```

I think the font-size should be `28px`?

Y5pUYSJPsGEud1vknUZhi8-fi-739_56524

```suggestion
		font-size: 28px;
		font-weight: 500;
```